tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -79,6 +80,55 @@ def set_certificate_in_satellite(server_type):
         raise AssertionError(f'Failed to restart the httpd after applying {server_type} cert')
 
 
+def unsubscribe():
+    """unregisters a machine from cdn"""
+    run_command('subscription-manager unregister')
+    run_command('subscription-manager clean')
+
+
+@pytest.fixture(scope='module')
+def clean_rhsm():
+    """removes pre-existing candlepin certs and resets RHSM."""
+    # removing the katello-ca-consumer
+    run_command('rpm -qa | grep katello-ca-consumer | xargs -r rpm -e')
+
+    # resetting rhsm.conf to point to cdn.
+    run_command(
+        "sed -i -e 's/^hostname.*/hostname=subscription.rhsm.redhat.com/' " "/etc/rhsm/rhsm.conf"
+    )
+    run_command("sed -i -e 's|^prefix.*|prefix=/subscription|' /etc/rhsm/rhsm.conf")
+    run_command("sed -i -e 's|^baseurl.*|baseurl=https://cdn.redhat.com|' " "/etc/rhsm/rhsm.conf")
+    run_command(
+        "sed -i -e "
+        "'s/^repo_ca_cert.*/repo_ca_cert=%(ca_cert_dir)sredhat-uep.pem/' "
+        "/etc/rhsm/rhsm.conf"
+    )
+
+
+@pytest.fixture(scope='module')
```

Comment:
don't we want this fixture to be session scoped ?

Reply:
let's keep it for module scope, for now,  we can change it to session scope whenever is in need.